### ISSUE TITLE
properties are not formatted

### ISSUE THREAD

**Issue by ubmarco** [nvuillam/vscode-groovy-lint#169]
I have this simple Jenkinsfile where properties are not properly indented. The formatter does nothing on the properties section:
```
properties(
            [
        parameters(
            [
                booleanParam(defaultValue: true, description: '', name: 'bool1'),
                booleanParam(defaultValue: false, description: '', name: 'bool2'),
            choice(choices: ['a', 'b', 'c'], description: '', name: 'choice1'),
                string(defaultValue: 'important input', description: '', name: 'string1', trim: false)
            ]
        ),
            ]
)

def printParams() {
    params.each { param, value ->
        print "Parameter: ${param}, Value: ${value}"
    }
}

stage('Check jenkins') {
    printParams()
}

```

The code in `def printParams() {` and `stage('Check jenkins') {` is properly indented, however.
Generally it seems like the formatter does not really deal well with brackets. Those are almost never touched.

**Comment by nvuillam** [comment]
formatting relies on IndentationRule CodeNarc rule results -> https://codenarc.org/codenarc-rules-formatting.html#indentation-rule

Unfortunately is does not catch all indentation issues, but npm-groovy-lint uses CodeNarc v3.10, maybe in 3.20 it will be better, we'll see once after npm-groovy-lint will be upgraded :/
